chain/chain/src/store_validator/validate.rs

```diff
@@ -906,23 +905,30 @@ pub(crate) fn block_height_cmp_tail_final(
 }
 
 pub(crate) fn gc_col_count_final(sv: &mut StoreValidator) -> Result<(), StoreValidatorError> {
```

Comment:
Add comment here for what this function does. Maybe we can also rename it to `check_final_gc_count`? Up to you

Reply:
Ok, that makes sense. Could you add a brief comment here to describe what this function does?